go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/conduitio/conduit-connector-kafka
+module github.com/conduitio-labs/conduit-connector-redpanda
 
 go 1.21.1
```

Comment:
Maybe we can use the chance and update this?

Reply:
+1 (go1.22.3 for reference)

Reply:
done